ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -204,8 +204,7 @@ extension MSFListView {
             Button(action: cell.onTapAction ?? {}, label: {
                 HStack(spacing: 0) {
                     if let leadingIcon = cell.leadingIcon {
-                        Image(uiImage: leadingIcon)
-                            .resizable()
+                        UIViewWrapper(leadingIcon)
```

Comment:
What would be the story for the app that is using SwiftUI fully?
I thought initially about having a modifier for the MSFListCellView where you'd pass "some View" (the AvatarVnext) in this case and it would replace the icon there.

And I believe that doesn't dismiss the current approach of supporting an instance of UIView.
I'm thinking about a path for both consumers (UIKit and SwiftUI) to use it.

Reply:
How would this be translated when declaring the input type? I understand that we can assign/return a `some View` type in SwiftUI but I do not think it can be represented in ObjC?

Reply:
https://github.com/microsoft/fluentui-apple/issues/380